test/datasets/test_molecule_gpt_dataset.py

```diff
@@ -8,3 +10,7 @@ def test_molecule_gpt_dataset():
     assert str(dataset) == f'MoleculeGPTDataset({len(dataset)})'
     assert dataset.num_edge_features == 4
     assert dataset.num_node_features == 6
+
+    # Optional cleanup if distributed is initialized
+    if dist.is_initialized():
+        dist.destroy_process_group()
```

Comment:
how about moving to `test/conftest.py`

Reply:
Done. 
@xnuohz: Thanks, it was a good suggestion!